apps/condo/pages/ticket/index.tsx

```diff
@@ -77,9 +77,8 @@ export const TicketsPageContent = ({
     const router = useRouter()
     const { filters } = parseQuery(router.query)
 
-    const reduceNonEmpty = (cnt, filter) => cnt + Number(Array.isArray(filters[filter]) && filters[filter].length > 0)
+    const reduceNonEmpty = (cnt, filter) => cnt + Number(filters[filter] && filters[filter].length > 0)
```

Comment:
In the case of filters[filter] = `<some Number>` you will get `0` instead of `1`. It means no button to reset filters.
<img width="281" alt="image" src="https://user-images.githubusercontent.com/25384290/176622383-bad36b31-3f78-4931-8513-fa187046b2d0.png">

Reply:
There is no case in ticket filters when the filter is a number.